.agents/skills/spec/reference/templates/tech.md

```
@@ -0,0 +1,115 @@
+
+## Design Philosophy
+
+1. **{Principle}.** {Technical guiding rule that constrains decisions.}
+2. **{Principle}.** {Technical guiding rule.}
+3. **{Principle}.** {Technical guiding rule.}
+
+---
+
+## Architecture Overview
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)